README.rst

```diff
@@ -92,3 +92,38 @@ __ https://docs.python.org/3/library/functools.html#functools.partial
 
         def handle(self, *args, **options):
             ...
+
+``django_rich.test.RichRunner``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A subclass of Django's |DiscoverRunner|__ with colourized outputs and `nice traceback rendering <https://rich.readthedocs.io/en/stable/traceback.html>`__.
+
+.. image:: img/RichRunner.png
+
+.. |DiscoverRunner| replace:: ``DiscoverRunner``
+__ https://docs.djangoproject.com/en/stable/topics/testing/advanced/#defining-a-test-runner
+
+To use this class, point your |TEST_RUNNER|__ setting to it:
+
+.. |TEST_RUNNER| replace:: ``TEST_RUNNER``
+__ https://docs.djangoproject.com/en/stable/ref/settings/#std:setting-TEST_RUNNER
+
+.. code-block:: python
+
+    TEST_RUNNER = "django_rich.test.RichRunner"
+
+You can also use it as a base for further customization.
+Since only output is modified, it should combine well with other classes.
+
+The test runner provides the following features:
+
+* Output is colourized wherever possible.
+  This includes Rich’s default `highlighting <https://rich.readthedocs.io/en/stable/highlighting.html>`__ which will format numbers, quoted strings, URL’s, and more.
+
+* Failures and errors use Rich’s `traceback rendering <https://rich.readthedocs.io/en/stable/traceback.html>`__.
+  This displays the source code and local values per frame.
+  Each frame also shows the filename and line number, and on many terminals you can click the link to jump to the file at that position.
+
+* Output is also colourized when using the ``--debug-sql`` and ``--pdb`` flags.
```

Comment:
I was apprehensive about commenting on this but as the latest commit changed the spelling. I hope this is ok.

American English : Color -> colorized
British English : colour -> colourised. 

Colourized is mix of the two? 🤔

Reply:
Haha I'm awkward with that... I write with [Oxford spelling](https://en.wikipedia.org/wiki/Oxford_spelling)... Perhaps I could stop doing that and confusing both camps...

Reply:
I think your are perfectly entitled to use whichever style you like for your projects!

That's interesting, I've not heard of this before. Will have a read. 👍